src/audiences/edit/components/select-include.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const { __ } = wp.i18n;
+
+const StyledSelect = styled.select`
+	vertical-align: middle;
+
+	&:not(:hover, :focus) {
+		border: none;
+		background: none;
+		margin-right: 8px;
+	}
+`;
+
+const SelectInclude = ( { value, onChange, label = '', name } ) => {
+	const id = name.replace( /\W+/g, '-' ).replace( /^\W+.*?\W+$/, '' );
```

Comment:
This is a little magic; could we refactor into a well-named function?

Reply:
I removed the `<label>` tag in the end which this was for so I'll just remove this.